app/tests/components/Form/ApplicationFormStatus.test.tsx

```diff
@@ -65,7 +65,7 @@ describe('The application form', () => {
     componentTestingHelper.renderComponent();
 
     expect(screen.queryByText('Saving')).toBeNull();
-    expect(screen.getByText('Last saved: Jan. 1')).toBeInTheDocument();
+    expect(screen.getByText('Last saved: Jan 1')).toBeInTheDocument();
```

Comment:
Another slight change, but also affects copy so making sure we confirmed this adjustment with PO?
```diff
- Last saved: Jan. 1
+ Last saved: Jan 1
```

Reply:
I'm currently trying to update code set explicit settings for Luxon to use defaultLocale="en-CA" which (hopefully) will solve formatting issues and allow us to revert those slight changes

Reply:
Actually, `en-CA` short month format does not have dot. 
https://metacpan.org/dist/DateTime-Locale/view/lib/DateTime/Locale/en_CA.pod#Medium and https://github.com/moment/luxon/blob/master/docs/formatting.md  (`DATE_MED`)

Reply:
I don't personally have strong feelings about keeping/removing the dot either way, just making sure we've looped in PO in the migration to the new format.